python/pyarrow/io.pxi

```diff
@@ -849,6 +850,15 @@ def frombuffer(object obj):
     return pyarrow_wrap_buffer(buf)
 
 
+def frombuffer(object obj):
+    """
+    Deprecated alias for `py_buffer`.
+    """
+    warnings.warn("pa.frombuffer() is deprecated, use pa.py_buffer() instead",
+                  DeprecationWarning)
```

Comment:
I thought DeprecationWarning does not show up by default. FutureWarning instead?

Reply:
Not by default, though it will show up in interactive settings such as IPython. I can switch to FutureWarning if you prefer.